### ISSUE TITLE
Языки в чате

### ISSUE THREAD

**Issue by hunterdelyx1** [hunterdelyx1/km-freeminer-mods#1]
Фразы внутри сообщения, заключённые в [текст], должны алгоритмически превращаться в фразы вымышленного языка расы персонажа игрока.

Используя [nтекст], где n - целое число, можно говорить на языка другой расы.

При этом персонажи той же расы должны видеть оригинальное сообщение.

Пример:
**Что вводит игрок:** Общеизвестное  слово "картошка" на моём язык будет звучать "[картошка]", на языке многоуважаемых маннов он будет звучать "[1картошка]"

**Вывод для персонажа, что не знает язык:** Общеизвестное  слово "картошка" на моём язык будет звучать "хукаарпа", на языке многоуважаемых маннов он будет звучать "хурьпа"

**Вывод для персонажа, что знает язык:** Общеизвестное  слово "картошка" на моём язык будет звучать "хукаарпа", на языке многоуважаемых маннов он будет звучать "хурьпа"
(( Общеизвестное  слово "картошка" на моём язык будет звучать "картошка", на языке многоуважаемых маннов он будет звучать "картошка" ))

**Comment by hunterdelyx1** [comment]
Нет, мы передумали. Лучше одно сообщение — один язык. И использовать для этого команду. Иначе реализация будет слишком сложной, да и избыточной.

**Comment by sullome** [comment]
и вы решили прописать все слова? Серьёзно?

**Comment by hunterdelyx1** [comment]
Нет, процедурное получение слов из изначальных слов, одно из двух:
1) Рандомная генерация с семенем по изначальному слову (vegasd)
2) Преобразование изначального слова по каким-то правилам (hunterdelyx1)

Выбор из этих двух. Первое гораздо интереснее, но если в слове допущена ошибка, то окончательные слова будут различаться слишком сильно, во втором методе этой проблемы нет, но он скучноват)

**Comment by sullome** [comment]
@hunterdelyx1, что по этой issue осталось сделать?

**Comment by hunterdelyx1** [comment]
@sullome генератор/переводчик, @vegasd обещал помочь мне с этим позже.
